# RobotAction.srv

string action_name
---
bool success